lib/tasks/database_updates/backfill_publisher_selected_wallet.rake

```diff
@@ -0,0 +1,105 @@
+namespace :database_updates do
+  desc 'Backfill Publisher Selected Wallets'
+  task :backfill_publisher_selected_wallets => :environment do
+    def handle_gemini
+      now = Time.zone.now
+      gemini_to_set = Publisher.
+        includes([:gemini_connection]).
+        joins(:gemini_connection).
+        where('gemini_connections.is_verified = TRUE').
+        where('publishers.email IS NOT NULL').
```

Comment:
```suggestion
```

Reply:
If you don't have an email, does it matter? I don't mind changing it, but I remember it made a difference. Will have to look it up. Perhaps it's due to bad data in the DB.